### ISSUE TITLE
material-ui 1.0.0-beta.24 breaks next beta.1

### ISSUE THREAD

**Issue by MikeSuiter** [TeamWertarbyte/material-ui-chip-input#170]
When updating to material-ui 1.0.0-beta.24 chips no longer render the delete icon and throws this error.

```
proxyConsole.js:68 Warning: Unknown event handler property `onRequestDelete`. It will be ignored.
    in div (created by Chip)
    in Chip (created by withStyles(Chip))
    in withStyles(Chip) (created by ChipInput)
    in div (created by ChipInput)
    in div (created by FormControl)
    in FormControl (created by withStyles(FormControl))
    in withStyles(FormControl) (created by ChipInput)
    in ChipInput (created by withStyles(ChipInput))
    in withStyles(ChipInput) (created by App)
    in App
```

Here is a sandbox that demos it:

https://codesandbox.io/s/2nl11q01n

**Comment by leMaik** [comment]
Indeed, it needs to be changed to `onDelete`, it's a breaking change in material-ui 1.0.0-beta.24 :confused:

**Comment by leMaik** [comment]
Fixed in [1.0.0-beta.2](https://github.com/TeamWertarbyte/material-ui-chip-input/releases/tag/v1.0.0-beta.2), thanks for reporting this! :+1:
